src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColumnBtn = document.querySelector('.append-column');
+const removeColumnBtn = document.querySelector('.remove-column');
+let rowCount = table.rows.length;
+let columnCount = table.rows[0].cells.length;
+
+if (rowCount === 10) {
+  appendRowBtn.disabled = true;
+}
+
+if (rowCount === 2) {
+  removeRowBtn.disabled = true;
+}
+
+if (columnCount === 10) {
+  appendColumnBtn.disabled = true;
+}
+
+if (columnCount === 2) {
+  removeColumnBtn.disabled = true;
+}
```

Comment:
Use switch/case instead

Reply:
I deleted it. Whithout this part of code evrething working good.